### PR TITLE
Fix Parent Service Definition

### PR DESCRIPTION
**Pull request by RocKordier**

### CODE REVIEW

File: src/Oro/Bundle/DataGridBundle/Resources/config/mass_actions.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@ services:
 
     oro_datagrid.extension.mass_action.type.massedit:
         class: Oro\Bundle\DataGridBundle\Extension\MassAction\Actions\Widget\MassEdit\MassEditAction
-        parrent: oro_datagrid.extension.mass_action.type.window
+        parent: oro_datagrid.extension.mass_action.type.window
```
Comment (anyt) [inline src/Oro/Bundle/DataGridBundle/Resources/config/mass_actions.yml]: This definition is redundant as there is nothing to extend from `oro_datagrid.extension.mass_action.type.window` service it makes sense to delete parent definition at all.

Comment (RocKordier) [reply src/Oro/Bundle/DataGridBundle/Resources/config/mass_actions.yml]: Hey @anyt ,
okay, i doesn't looked deep into the code. 
I've removed the `parent` statement and force pushed the change.

Thx Eric

Comment (RocKordier) [reply src/Oro/Bundle/DataGridBundle/Resources/config/mass_actions.yml]: Nope, unfortunately not.
But I already discussed this kind of tool with @Baachi

I've found it in the profiler logs
screenshot 2018-12-11 at 17 18 59

Comment (anyt) [reply src/Oro/Bundle/DataGridBundle/Resources/config/mass_actions.yml]: Logs are enough to automate this check, thank you.

**Comment by RocKordier** [comment]
Committed in 2a14792824264cd7e437349a9ba52f840a903d37 
thx.
closed

**Pull request closed without merging**